### PR TITLE
Release note for signal forwarding deprecation

### PR DESCRIPTION
**Pull request by jpipkin1**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request adds a Cloud SIEM release note written by @pkazmir-sumo:
https://help.sumologic.com/release-notes-cse/2023/10/18/application-update/

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, would you be able to review and approve this PR? @pkazmir-sumo would like to get it out today.

**Review by pkazmir-sumo — APPROVED** [review]
Looks good to me

File: blog-cse/2023-10-18-application-update.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The new index is automatically generated and retained for a period of 2 years at no additional cost for all Cloud SIEM customers.
+
+As a result, the optional legacy Signal Forwarding feature in Cloud SIEM will be deprecated on **November 15, 2023**. Existing data will not be deleted, but new Signals generated after that date will no longer be forwarded using that feature and the option will no longer be available. (Signals will continue to be forwarded automatically to `sec_signal`.) Customers leveraging data forwarded using the legacy feature to generate dashboards (or for other use cases) will need to modify those applications to use the new `sec_signal` index before then. Note that the content of the `sec_signal` index is not identical to the content in data forwarded using the legacy option.
+
+For more information about this change, and the differences between the two data sets, please refer to our [FAQ document](/docs/cse/records-signals-entities-insights/signal-index-migration-faq/).
```
Comment (kimsauce) [inline blog-cse/2023-10-18-application-update.md:21]:
```suggestion
For more information about this change, and the differences between the two data sets, refer to our [2023 Cloud SIEM Signal Index Migration FAQ](/docs/cse/records-signals-entities-insights/signal-index-migration-faq/). 
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
had an SEO-boosting suggestion

**Comment by jpipkin1** [comment]
Thanks for the quick review, @kimsauce!

**Pull request merged**